lib/vsc/filesystem/oceanstor.py

```diff
@@ -1097,13 +1098,35 @@ def make_fileset(
             # wait for NFS lookup cache to expire to be able to access new fileset
             time.sleep(NFS_LOOKUP_CACHE_TIME)
 
-        # Set default user quota: 1MB for blocks soft limit and inodes_max for inodes hard limit
+        # Set default user quota on blocks soft limit and inodes hard limit from settings in VscStorage
         # TODO: remove once OceanStor supports setting user quotas on non-empty filesets
-        block_soft = DEFAULT_USER_BLOCK
-        inodes_soft = int(inodes_max // OCEANSTOR_QUOTA_FACTOR)
-        self.set_user_quota(block_soft, "*", obj=dtree_fullpath, inode_soft=inodes_soft)
-        grace_time = DEFAULT_GRACE_DAYS * 24 * 3600
-        self.set_user_grace(dtree_fullpath, grace=grace_time, who="*")
+        try:
+            vsc_fileset_storage = [
+                stor
+                for inst in self.institutes
+                for stor in self.vscstorage[inst].values()
+                if stor.backend == 'oceanstor' and dtree_fullpath.startswith(stor.backend_mount_point)
+            ][0]
+        except IndexError as err:
+            errmsg = "Could not find VSC storage for new fileset '%s' at: %s"
+            self.log.raiseException(errmsg % (ostor_dtree_name, ostor_parentdir), OceanStorOperationError)
+        else:
+            if ostor_dtree_name[1:3] == 'vo':
```

Comment:
don't we have in vsc-config something for this? VO_PREFIX?

Reply:
indeed, there is `VO_INFIX`